Tests/Algorithm/Framework/Alphas/CommonAlphaModelTests.cs

```diff
@@ -151,7 +151,10 @@ public void ModelNameTest(Language language)
             }
 
             var actual = model.GetModelName();
-            var expected = GetExpectedModelName(model);
+            var expected = language == Language.CSharp
+                ? GetExpectedModelName(model)
+                : (model as AlphaModelPythonWrapper).Name;
```

Comment:
for python, won't this _always_ be true? maybe the `GetExpectedModelName` implementation should check `if (model is PyObject)` or something similar?

Reply:
Please update the python alpha models to implement the `Name` property so that it equals the c# version Name properties. While I don't expect many users to define names for their models, we still need the examples in python for how it's done, I'm hoping it's simple, if not, we may need to do a refactoring to make it easier for python users.